setup_wizard/character_rig_setup/rig_script.py

```diff
@@ -2949,7 +2962,17 @@ def bone_to_layer(bone, layer, collection, second_coll="None"):
     bone_to_layer("f_pinky.03.R", 6, "Fingers (Detail)")  
     bone_to_layer("f_pinky.01.L.001", 6, "Fingers (Detail)")  
     bone_to_layer("f_pinky.01.R.001", 6, "Fingers (Detail)") 
-    
+
+    bone_to_layer("Lighting Panel", 1, "Lighting")
+    bone_to_layer("Fresnel", 1, "Lighting")
+    bone_to_layer("Ambient", 1, "Lighting")
+    bone_to_layer("SoftLit", 1, "Lighting")
+    bone_to_layer("Lit", 1, "Lighting")  # Sharp Lit
+    bone_to_layer("SoftShadow", 1, "Lighting")
+    bone_to_layer("Shadow", 1, "Lighting")  # Sharp Shadow
+    bone_to_layer("RimShadow", 1, "Lighting")
+    bone_to_layer("Rim Lit", 1, "Lighting")

```

Comment:
Arbitrarily chose Layer 1, but we can change it if desired

Reply:
shouldnt matter, any layer num could be used so long as it hasnt been used before. until we drop support for 3.6 we'll be tied down to only the numbers made available with the old bone layer system.